### PR TITLE
Added gateway

### PR DESCRIPTION
**Pull request by kjuulh**

### CODE REVIEW

File: gateway/Gateway/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": false,
+      "launchUrl": "api/health/ping",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "PSO_Control_Service": {
```
Comment (jakob12317) [inline gateway/Gateway/Properties/launchSettings.json]: Ikke PSO_Control_service

File: services/authentication/Authentication/Properties/launchSettings.json [hunk 2]
```diff
@@ -12,24 +12,26 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "api/UserController",
+      "launchUrl": "api/health/ping",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
     "PSO_Control_Service": {
```
Comment (jakob12317) [inline services/authentication/Authentication/Properties/launchSettings.json]: Samme her.

File: gateway/Gateway/Properties/launchSettings.json (hunk 1 above)
Comment (kjuulh) [reply gateway/Gateway/Properties/launchSettings.json]: fixed

File: services/authentication/Authentication/Properties/launchSettings.json (hunk 2 above)
Comment (kjuulh) [reply services/authentication/Authentication/Properties/launchSettings.json]: fixed

**Pull request merged**